config/source/common/serviceaccount.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 The Knative Authors
+# Copyright 2019 The Knative Authors
```

Comment:
hrm.... ?

Reply:
Looks like a Git glitch. It was probably another file that got wrongly considered as "renamed".